virttest/qemu_vm.py

```diff
@@ -906,6 +936,8 @@ def add_pcidevice(devices, host, params, device_driver="pci-assign",
                        " It only support following parameter:\n %s" %
                        (", ".join(fail_param), pcidevice_help))
                 LOG.warn(msg)
+            set_cmdline_format_by_cfg(dev, self._get_cmdline_format_cfg(),
+                                      "images")
```

Comment:
The `pcidevice`/`vfio-pci` device is not a kind of images, could you please double confirm that?

Reply:
> The `pcidevice`/`vfio-pci` device is not a kind of images, could you please double confirm that?

After I check the xml file, I may find "controller". I'm NOT sure which I use at here, "images" or "controller"?